src/ansys/fluent/core/services/field_data.py

```diff
@@ -634,7 +634,7 @@ def _get_surface_ids(
         surface_ids = []
         if surface_names:
             for surface_name in surface_names:
-                surface_ids.append(
+                surface_ids.extend(
```

Comment:
Will add a proper test shortly but this is a fixed use case - first call raises without the change.

```
>>> data_trans.add_scalar_fields_request(field_name='z-velocity', surface_names=["wall", "pressure-outlet-7"])
>>> dir(data_trans)
['__call__', '__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_allowed_scalar_field_names', '_allowed_surface_names', '_allowed_vector_field_names', '_field_info', '_fields_request', '_service', 'add_pathlines_fields_request', 'add_scalar_fields_request', 'add_surfaces_request', 'add_vector_fields_request', 'get_fields']
>>> data_trans.get_fields()
{(('type', 'scalar-field'), ('dataLocation', 0), ('boundaryValues', False)): {5: {'z-velocity': array([ 0.00949505,  0.01040305,  0.00948992, ..., -0.00566791,
       -0.00536598, -0.00503718], dtype=float32)}, 3: {'z-velocity': array([ 8.09024932e-05,  8.86667658e-06,  2.95718837e-05,  1.50164822e-04,

```